index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
```

Comment:
Sometimes I respond too fast for the page to update! If you perform an expected action and don't see a response, wait a few seconds and refresh the page for your next steps.